deploy/ansible/roles/goshimmer-node/templates/docker-compose-public-node.yml.j2

```diff
@@ -0,0 +1,65 @@
+#jinja2: trim_blocks:True, lstrip_blocks:True
```

Comment:
I like these different compose files, even if there is a lot of duplication: it makes the content easier to read with less ifs and such.

Reply:
Yeah I tried to balance between readability and amount of duplication.